apps/consumer-client/src/pages/examples/add-pcd.tsx

```diff
@@ -110,45 +110,46 @@ function AddEthAddrPCDButton(): JSX.Element {
 
     const parsed = JSON.parse(pcdStr) as SerializedPCD;
 
-    const ethereum = (window as any).ethereum;
-    const provider = new ethers.providers.Web3Provider(ethereum);
-    if (!ethereum) {
+    if (!("ethereum" in window)) {
```

Comment:
This avoids checking the value of `(window as any).ethereum`.

Putting the async function in an `else` block seems like a good idea since it depends on `window.ethereum`.